src/components/header/HeaderComponent.vue

```diff
@@ -0,0 +1,63 @@
+<script setup>
+import { ref } from 'vue';
+import { RouterLink } from 'vue-router';
+
+import jazzHeader from '@/assets/images/Jazzheader.png';
+import danceHeader from '@/assets/images/Danceheader.png';
+import yummyHeader from '@/assets/images/Yummyheader.png';
+import homeHeader from '@/assets/images/Homeheader.png';
+import historyHeader from '@/assets/images/Historyheader.png';
+import shopHeader from '@/assets/images/Shopheader.png';
+
+import userHeader from '@/assets/images/Userheader.png';
+import languageHeader from '@/assets/images/Languageheader.png';
+
+const showUserDropdown = ref(false);
+</script>
+
+<template>
+  <body>
+    <div id="app">
+      <header class="header">
+        <nav>
+          <div class="links-container">
+            <div class="user-container">
+              <RouterLink to="/user" class="header-link user" @click.prevent="showUserDropdown = !showUserDropdown">
```

Comment:
zorg ervoor dat de click tijdens een hover/mouseover event: https://www.w3schools.com/jsref/event_onmouseover.asp 

anders blijft het inbeeld en dat is niet zo netjes